modules/backend/widgets/search/partials/_search.htm

```diff
@@ -3,18 +3,17 @@
         placeholder="<?= $placeholder ?>"
         type="text"
         name="<?= $this->getName() ?>"
-        value="<?= $value ?>"
+        value="<?= $value ?>"		
         data-request="<?= $this->getEventHandler('onSubmit') ?>"
+		data-request-complete="if($(this).val().length) {$(this).next().show()} else {$(this).next().hide()};"
```

Comment:
Use spaces, not tabs - and please remove trailing whitespace

Reply:
hopefully also done